WordPress/Classes/Models/Blog+Post.swift

```diff
@@ -28,3 +30,82 @@ public extension Blog {
         return (try? context.fetch(request))?.first
     }
 }
+
+// MARK: - Create posts
+
+extension Blog {
+
+    /// Create a post in the blog.
+    @objc
+    func createPost() -> Post {
+        guard let context = managedObjectContext else {
+            fatalError("The `Blog` instance is not associated with a `NSManagedObjectContext`")
```

Comment:
Grammar nitpick. It is my understanding that, since the "N" in "NSManaged..." is part of the "NS (NextStep)" acronym, and is therefore read "en es managed ..." the article to use for it should be "an".

```suggestion
            fatalError("The `Blog` instance is not associated with an `NSManagedObjectContext`")
```

Sources [this](https://www.grammar.com/a-vs-an-when-to-use) and [this](https://www.quora.com/Do-we-use-an-before-words-starting-with-n)

Reply:
Good catch!